ray-operator/controllers/ray/common/constant.go

```diff
@@ -9,10 +9,14 @@ const (
 	RayNodeLabelKey                    = "ray.io/is-ray-node"
 	RayIDLabelKey                      = "ray.io/identifier"
 	RayClusterDashboardServiceLabelKey = "ray.io/cluster-dashboard"
+	RayClusterServingServiceLabelKey   = "ray.io/serving"
 
 	EnableAgentServiceKey  = "ray.io/enableAgentService"
 	EnableAgentServiceTrue = "true"
 
+	RayClusterServingServiceTrue  = "true"
```

Comment:
You mean heathy true or false or something else? Can we make it self-explainable?